org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/Types.java

```diff
@@ -193,19 +193,19 @@ public FunctionBreakpoint(String name) {
     public static class ExceptionBreakpointFilter {
         public static final String Filter_All = "all";
         public static final String Filter_UserUnhandled = "user-unhandled";
+        public static final String Filter_UserHandled = "user-handled";
 
         public String label;
         public String filter;
-
-        /**
-         * Constructor.
-         */
-        public void setFilter(String value) {
-            if (!Filter_All.equals(value) && !Filter_UserUnhandled.equals(value)) {
-                return;
-            }
+        
+        public ExceptionBreakpointFilter(String value, String label) {
             this.filter = value;
+            this.label = label;
         }
+        
+        public static ExceptionBreakpointFilter All = new ExceptionBreakpointFilter(Filter_All, "All Exceptions");
```

Comment:
i found some discussion [here](https://github.com/Microsoft/vscode/issues/11552). right now we don't support exceptionoptions